src/main/java/seedu/address/storage/XmlAdaptedCard.java

```diff
@@ -71,6 +74,11 @@ public Card toModelType() throws IllegalValueException {
         if (!Card.isValidCard(this.back)) {
             throw new IllegalValueException(Card.MESSAGE_CARD_CONSTRAINTS);
         }
+        if (FillBlanksCard.containsBlanks(this.front)) {
```

Comment:
this looks to be very similar to a normal card. The only reason I can think of for having a special card for this is when we want to reveal the blanks one at a time.

Reply:
You mean the implementation or you mean there's no need for fill-blanks card?

Reply:
Both. The similarities mean that there's possibility no need for special handling to get this kind of card differentiation. e.g. you could denote _ to be blanks and there would be no need for a new parser which supports multiple fronts and backs.